grails-plugin-controllers/src/main/groovy/grails/artefact/controller/support/ResponseRenderer.groovy

```diff
@@ -118,7 +119,7 @@ trait ResponseRenderer extends WebAttributes {
      * @param closure The markup to render
      */
     @Generated
-    void render(Closure closure) {
+    void render(@DelegatesTo(strategy = Closure.DELEGATE_FIRST) Closure closure) {
```

Comment:
Could you please share why the delegate strategy is removed?

Reply:
So it was either add that, or Skip static compilation on that method.  I am not sure the consequences of changing the strategy, but it was the only way I could get it to compile and pass the unit tests.

Reply:
Thank you for clarification. I think the delegate strategy should be marked to each method in the chain which should resolve the issue. I will clarify further with Paul regarding this.